.gitignore

```diff
@@ -21,6 +21,13 @@ coverage/
 # Gemfile.lock
 
 .idea/
+routes_list.rb
+/public/*
+
+# Ignore all files ending with ds_store
+.DS_Store
```

Comment:
.DS_Store is already recursive i.e. it removes all DS_Store files from the project. You don't need a config/.DS_Store